Makefile

```diff
@@ -84,7 +84,7 @@ docker-push:
 # download controller-gen if necessary
 controller-gen:
 ifeq (, $(shell which controller-gen))
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.0-beta.2
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4
```

Comment:
The CRD in path `./config/crd/bases/` doesn't seem to be generated using this version.
Please keep in mind that the CRD generated using this version will not be compatible with k8s versions < 1.15 mainly because of an extra added field `.spec.preserveUnknownFields`